### PR TITLE
feat(replays): New column size and improve of a11y

### LINKED ISSUE #37615: Network Table should have a "Resource Size" column

### PR DESCRIPTION
**Pull request by jesus4497**
### Changes
- Added new column `size` to the network table
- Improved a11y and some styles

Closes #37615 

image

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: static/app/views/replays/detail/network/index.tsx [hunk 1]
```diff
@@ -144,6 +157,14 @@ function NetworkList({replayRecord, networkSpans}: Props) {
         <Item {...columnHandlers}>
           <Text>{network.op.replace('resource.', '')}</Text>
         </Item>
+        <Item {...columnHandlers} numeric>
+          {network.data.size ? (
+            `${network.data.size} B`
+          ) : (
```
Comment (billyvg) [inline static/app/views/replays/detail/network/index.tsx:163]: We have a component or function that formats this nicely.. it should be on storybook

Comment (jesus4497) [reply static/app/views/replays/detail/network/index.tsx]: Fixed, updated the description with the new capture

File: static/app/views/replays/detail/network/index.tsx [hunk 2]
```diff
@@ -144,6 +158,14 @@ function NetworkList({replayRecord, networkSpans}: Props) {
         <Item {...columnHandlers}>
           <Text>{network.op.replace('resource.', '')}</Text>
         </Item>
+        <Item {...columnHandlers} numeric>
+          {network.data.size ? (
```
Comment (billyvg) [inline static/app/views/replays/detail/network/index.tsx:162]: `0` will incorrectly show as missing

**Review by billyvg — APPROVED** [review]
LGTM!

**Pull request merged**